test/helpers/web3/getWeb3.js

```diff
@@ -0,0 +1,21 @@
+const Ganache = require(process.env.TEST_BUILD
+  ? "../build/ganache.core." + process.env.TEST_BUILD + ".js"
+  : "../../../index.js");
+const Web3 = require("web3");
+
+const getWeb3 = async(options = {}) => {
```

Comment:
This doesn't return a web3 instance - it returns an object which contains a web3 instance. perhaps it should be `initializeTestProvider` or something like that?

Also bear in mind that we'll need the ability to initialize tests which operate using both remote and local providers.

Reply:
Good change request.

Reply:
Module and filename changed to `initializeTestProvider`.